package.json

```diff
@@ -32,13 +32,13 @@
     "socket.io-client": "^4.2.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.14.8",
-    "@babel/core": "^7.15.0",
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.5",
```

Comment:
I think these conflicts would go away if you used nvm. In the root dir, type `nvm use` and then do `npm i` again